tests/merlincat/event_packer.c

```diff
@@ -470,7 +470,11 @@ merlin_event *event_packer_unpack_kvv(const char *cmd, struct kvvec *kvv) {
 		res = kvvec_to_adaptive_contact(kvv, unpacked_data);
 		break;
 	case CTRL_PACKET:
-		evt->hdr.code = CTRL_ACTIVE; /* Todo: Assume CTRL_ACTIVE */
+		if (0 == strcmp("CTRL_INVALID_CLUSTER", cmd)) {
+			evt->hdr.code = CTRL_INVALID_CLUSTER;
+		} else {
+			evt->hdr.code = CTRL_ACTIVE; /* Todo: Assume CTRL_ACTIVE */
```

Comment:
Just leaving a comment here to make sure you left this "todo" on purpose :)

Reply:
It was there from before, so not really sure what the intention were. Afaik we don't use any other CTRL codes anymore (expect the ones recently implemented), so I got rid of the TODO.